docs/hub/spaces-sdks-static.md

```diff
@@ -6,7 +6,25 @@ Here are some examples of Spaces using custom HTML:
 
 * [Smarter NPC](https://huggingface.co/spaces/mishig/smarter_npc): Display a PlayCanvas project with an iframe in Spaces.
 * [Huggingfab](https://huggingface.co/spaces/pierreant-p/huggingfab): Display a Sketchfab model in Spaces.
+* [Diffuse the rest](https://huggingface.co/spaces/huggingface-projects/diffuse-the-rest): Draw and diffuse the rest
 
+## Adding a build step before serving
+
+Static Spaces support adding a custom build step before serving your static assets. This is useful for frameworks like Next.js, SvelteKit, and others that require a build process before serving. The build command runs automatically when your Space is updated.
```

Comment:
note that Next.js and sveltekit may not be the most appropriate as they specialize in SSR

Reply:
mhh it should work with adapter static (didn't check the code)

Reply:
if it doesnt work for next.js/sveltekit we can rollaback the wording to react/svelte/vue